getJobs.js

```diff
@@ -0,0 +1,53 @@
+module.exports.getJobs = function getJobs(requestedGroupID) {
+  return new Promise((resolve) => {
```

Comment:
You don't need to create a new promise in this way. We only `new Promise` when transforming non-promise functions to promise.